### PR TITLE
[#151007160] Show login form

### PR DESCRIPTION
**Pull request by bbmoz**
This PR introduces a number of things:
1. [CSS Grid](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Grid_Layout)
1. [CSS Modules](https://github.com/css-modules/css-modules)
1. [React](https://facebook.github.io/react/docs/hello-world.html)
1. [Webpack](https://webpack.js.org/)
1. Testing with [Ava](https://github.com/avajs/ava) and [Supertest](https://github.com/visionmedia/supertest)

This PR is part of the following story:
https://www.pivotaltracker.com/story/show/151007160

This is not yet complete until the following story is done:
https://www.pivotaltracker.com/story/show/150994053

### CODE REVIEW

**Comment by bbmoz** [comment]
@fybwid The `index.html` location is now different because we will need to run the build with webpack before the server is run. Run `yarn start` and `yarn start:dev` and notice the `client/dist/`. See *package.json* scripts for details.

File: server/index.js [hunk 1]
```diff
@@ -1,10 +1,14 @@
 const express = require('express')
+const path = require('path')
 const router = require('./router')
 
 const app = express()
+app.use(express.static(path.join(__dirname, '..', 'client', 'dist')))
```
Comment (fybwid) [inline server/index.js]: Possible action 2:
Change `app.use(express.static(path.join(__dirname, '..', 'client', 'dist')))` to `app.use(express.static(path.join(__dirname, '..', 'client', 'src')))`

Comment (bbmoz) [reply server/index.js]: Thanks. There is a build step that occurs. See previous comment.

**Comment by sahithyakatta** [comment]
Every thing looks good for me.

**Pull request merged**